cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -433,9 +442,5 @@ func (o *Options) ApplyTo(config *apiserver.Config) error {
 		),
 	}
 
-	if err := o.Recommended.Etcd.Complete(config.GenericConfig.StorageObjectCountTracker, config.GenericConfig.DrainedNotify(), config.GenericConfig.AddPostStartHook); err != nil {
-		return err
-	}
```

Comment:
Where is this gone/how is this replaced? 👀

Reply:
It is not done inside `ApplyWithStorageFactoryTo`. ref - https://github.com/kubernetes/kubernetes/pull/118416

Reply:
No, this is already passed in `ApplyWithStorageFactoryTo` second argument as part of `gardenerAPIServerConfig.Config`